docs/technical_reference/property_models/ASM2D_N2O.rst

```diff
@@ -0,0 +1,342 @@
+
+:Note: Parameter values found in asm2dinit_bsm2.c: https://app.box.com/file/1812331425408
+
+.. csv-table::
+   :header: "Description", "Symbol", "Parameter", "Default Value", "Units"
+
+   "C content of inert soluble COD, S_I", ":math:`i_{CSI}`", "i_CSI", 0.36178, ":math:`\text{dimensionless}`"
+   "C content of soluble COD, S_F", ":math:`i_{CSF}`", "i_CSF", 0.31843, ":math:`\text{dimensionless}`"
+   "C content of acetate, S_A", ":math:`i_{CSA}`", "i_CSA", 0.375, ":math:`\text{dimensionless}`"
+   "C content of inert particulate COD, X_I", ":math:`i_{CXI}`", "i_CXI", 0.36178, ":math:`\text{dimensionless}`"
+   "C content of particulate COD, X_S", ":math:`i_{CXS}`", "i_CXS", 0.31843, ":math:`\text{dimensionless}`"
+   "C content of biomass", ":math:`i_{CBM}`", "i_CBM", 0.36612, ":math:`\text{dimensionless}`"
+   ":lime:`C content, X_PHA`", ":math:`i_{CXPHA}`", "i_CXPHA", 0.3, ":math:`\text{dimensionless}`"
+   "N content of inert soluble, COD S_I", ":math:`i_{NSI}`", "i_NSI", 0.06003, ":math:`\text{dimensionless}`"
+   "N content of soluble COD, S_F", ":math:`i_{NSF}`", "i_NSF", 0.03552, ":math:`\text{dimensionless}`"
+   "N content of inert particulate COD, X_I", ":math:`i_{NXI}`", "i_NXI", 0.06003, ":math:`\text{dimensionless}`"
+   "N content of particulate COD X_S", ":math:`i_{NXS}`", "i_NXS", 0.03552, ":math:`\text{dimensionless}`"
+   "N content of biomass, X_H, X_PAO, X_AOB, X_NOB", ":math:`i_{NBM}`", "i_NBM", 0.08615, ":math:`\text{dimensionless}`"
+   ":blue:`P content of inert soluble COD S_I`", ":math:`i_{PSI}`", "i_PSI", 0.00649, ":math:`\text{dimensionless}`"
+   "P content of soluble COD, S_F", ":math:`i_{PSF}`", "i_PSF", 0.00559, ":math:`\text{dimensionless}`"
+   "P content of inert particulate COD, X_I", ":math:`i_{PXI}`", "i_PXI", 0.00649, ":math:`\text{dimensionless}`"
+   "P content of particulate COD, X_S", ":math:`i_{PXS}`", "i_PXS", 0.00559, ":math:`\text{dimensionless}`"
+   "P content of biomass, X_H, X_PAO, X_AOB, X_NOB", ":math:`i_{PBM}`", "i_PBM", 0.02154, ":math:`\text{dimensionless}`"
+   "TSS to COD ratio for X_I", ":math:`i_{TSSXI}`", "i_TSSXI", 0.75, ":math:`\text{dimensionless}`"
+   "TSS to COD ratio for X_S", ":math:`i_{TSSXS}`", "i_TSSXS", 0.75, ":math:`\text{dimensionless}`"
+   "TSS to COD ratio for biomass, X_H, X_PAO, X_AUT", ":math:`i_{TSSBM}`", "i_TSSBM", 0.90, ":math:`\text{dimensionless}`"
+   "Fraction of S_I from hydrolysis", ":math:`f_{SI}`", "f_SI", 0, ":math:`\text{dimensionless}`"
+   "Yield of heterotrophic biomass, X_H", ":math:`Y_{H}`", "Y_H", 0.625, ":math:`\text{dimensionless}`"
+   "Fraction of inert COD from lysis", ":math:`f_{XI}`", "f_XI", 0.1, ":math:`\text{dimensionless}`"
+   "Fraction of inert COD from lysis", ":math:`f_{XIH}`", "f_XIH", 0.1, ":math:`\text{dimensionless}`"
+   "Fraction of inert COD from lysis", ":math:`f_{XIP}`", "f_XIP", 0.1, ":math:`\text{dimensionless}`"
+   "Fraction of inert COD from lysis", ":math:`f_{XIA}`", "f_XIA", 0.1, ":math:`\text{dimensionless}`"
```

Comment:
@adam-a-a Just to provide a bit of context here, these are actually not mistranslations - the c-code provides the same description for all of these variables (f_XI is technically blank, but this description can be seen in other files). Nevertheless, we prob should add some descriptor to clearly distinguish these parameters from one another